### PR TITLE
Docker small fixes

### PR DESCRIPTION
**Pull request by KrisThielemans**
see commit messages if what this does.

@paskino, could you have a quick look?

### CODE REVIEW

File: docker/build_system-ubuntu.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@ set -ev
 # SIRF external dependencies
 apt-get update -qq
 apt-get install -yq --no-install-recommends \
-  libboost-all-dev     \
+  libboost-dev libboost-chrono-dev \
+  libboost-filesystem-dev libboost-thread-dev \
+  libboost-date-time-dev libboost-regex-dev \
+  libboost-program-options-dev libboost-atomic-dev \
+  libboost-test-dev libboost-timer-dev \
```
Comment (paskino) [inline docker/build_system-ubuntu.sh:10]: didn't we decide to remove these packages?

**Review by paskino — APPROVED** [review]
this should be fine

File: docker/build_system-ubuntu.sh (hunk 1 above)
Comment (KrisThielemans) [reply docker/build_system-ubuntu.sh:10]: I copied the list from the VM. Not 100% sure if we still need all of them, but definitely most of them.

The previous version installed all the boost libraries, which is considerably more

**Comment by KrisThielemans** [comment]
thanks. feel free to merge (*not* squash-merge), or tell me to do it

**Pull request merged**